snuba/util.py

```diff
@@ -15,7 +15,7 @@
 import re
 import simplejson as json
 import six
-import _strptime  # fixes _strptime deferred import issue
+import _strptime  # NOQA fixes _strptime deferred import issue
```

Comment:
@alex-hofsteede how do deferred imports happen here? Added a noqa to keep the linter from complaining.

Reply:
IIRC the first time we use `datetime.strptime` it internally imports `_strptime` and somehow that can cause an issue at runtime so we make sure it's imported up front, but I don't remember all the details.